--- mailgraph-0.20/mailgraph.cgi	Tue Jul  2 11:14:49 2002
+++ mailgraph-0.20.dzikus/mailgraph.cgi	Tue Jul 30 14:12:48 2002
@@ -10,13 +10,13 @@
 my $VERSION = "1.6";
 
 my $host = (POSIX::uname())[1];
-my $scriptname = 'mailgraph.cgi';
+my $scriptname = 'index.cgi';
 my $xpoints = 540;
 my $points_per_sample = 3;
 my $ypoints = 150;
 my $ypoints_err = 80;
-my $rrd = 'mailgraph.rrd'; # path to where the RRD database is
-my $rrd_virus = 'mailgraph_virus.rrd'; # path to where the Virus RRD database is
+my $rrd = '/var/lib/mailgraph/mailgraph.rrd'; # path to where the RRD database is
+my $rrd_virus = '/var/lib/mailgraph/mailgraph_virus.rrd'; # path to where the Virus RRD database is
 my $tmp_dir = '/tmp/mailgraph'; # temporary directory where to store the images
 
 my @graphs = (
--- mailgraph-1.2/mailgraph.pl.orig	Sun Jan  5 23:32:04 2003
+++ mailgraph-1.2/mailgraph.pl	Tue Jan 28 12:21:21 2003
@@ -226,8 +226,8 @@
 
 # global variables
 my $logfile;
-my $rrd = "mailgraph.rrd";
-my $rrd_virus = "mailgraph_virus.rrd";
+my $rrd = "/var/lib/mailgraph/mailgraph.rrd";
+my $rrd_virus = "/var/lib/mailgraph/mailgraph_virus.rrd";
 my $year;
 my $this_minute;
 my %sum = ( sent => 0, received => 0, bounced => 0, rejected => 0, virus => 0, spam => 0 );
@@ -253,7 +253,7 @@
 	print "  -v, --verbose      be verbose about what you do\n";
 	print "  -V, --version      output version information and exit\n";
 	print "  -c, --cat          causes the logfile to be only read and not monitored\n";
-	print "  -l, --logfile f    monitor logfile f instead of /var/log/syslog\n";
+	print "  -l, --logfile f    monitor logfile f instead of /var/log/maillog\n";
 	print "  -y, --year         starting year of the log file (default: current year)\n";
 	print "      --host=HOST    use only entries for HOST (regexp) in syslog\n";
 	print "  -d, --daemon       start in the background\n";
@@ -286,7 +286,7 @@
 	$daemon_rrd_dir = $opt{daemon_rrd} if defined $opt{daemon_rrd};
 	daemonize if $opt{daemon};
 
-	my $logfile = defined $opt{logfile} ? $opt{logfile} : '/var/log/syslog';
+	my $logfile = defined $opt{logfile} ? $opt{logfile} : '/var/log/maillog';
 	my $file;
 	if($opt{cat}) {
 		$file = $logfile;
