diff -urN mailgraph-0.20/mailgraph.cgi mailgraph-0.20.dzikus/mailgraph.cgi
--- mailgraph-0.20/mailgraph.cgi	Tue Jul  2 11:14:49 2002
+++ mailgraph-0.20.dzikus/mailgraph.cgi	Tue Jul 30 14:12:48 2002
@@ -14,8 +14,8 @@
 my $xpoints = 540;
 my $ypoints = 150;
 my $ypoints_err = 80;
-my $rrd = 'mailgraph.rrd'; # path to where the RRD database is
-my $rrd_virus = 'mailgraph_virus.rrd'; # path to where the Virus RRD database is
+my $rrd = '/var/lib/mailgraph/mailgraph.rrd'; # path to where the RRD database is
+my $rrd_virus = '/var/lib/mailgraph/mailgraph_virus.rrd'; # path to where the Virus RRD database is
 my $tmp_dir = '/tmp/mailgraph'; # temporary directory where to store the images
 
 my @graphs = (
diff -urN mailgraph-0.20/mailgraph.pl mailgraph-0.20.dzikus/mailgraph.pl
--- mailgraph-0.20/mailgraph.pl	Tue Jul  2 09:31:11 2002
+++ mailgraph-0.20.dzikus/mailgraph.pl	Tue Jul 30 14:12:11 2002
@@ -195,8 +195,8 @@
 
 # global variables
 my $logfile;
-my $rrd = "mailgraph.rrd";
-my $rrd_virus = "mailgraph_virus.rrd";
+my $rrd = "/var/lib/mailgraph/mailgraph.rrd";
+my $rrd_virus = "/var/lib/mailgraph/mailgraph_virus.rrd";
 my $year;
 my $this_minute;
 my %sum = ( sent => 0, received => 0, bounced => 0, rejected => 0, virus => 0, spam => 0 );
@@ -217,7 +217,7 @@
 {
 	print "usage: mailgraph [*options*]\n\n";
 	print "  -c, --cat          causes the logfile to be only read and not monitored\n";
-	print "  -l, --logfile f    monitor logfile f instead of /var/log/syslog\n";
+	print "  -l, --logfile f    monitor logfile f instead of /var/log/maillog\n";
 	print "  -h, --help         display this help and exit\n";
 	print "  -v, --version      output version information and exit\n";
 	print "  -y, --year         starting year of the log file (default: current year)\n";
@@ -237,7 +237,7 @@
 		exit;
 	}
 
-	my $logfile = defined $opt{logfile} ? $opt{logfile} : '/var/log/syslog';
+	my $logfile = defined $opt{logfile} ? $opt{logfile} : '/var/log/maillog';
 	my $file;
 	if($opt{cat}) {
 		$file = $logfile;
